feature/account/setup/src/main/kotlin/app/k9mail/feature/account/setup/ui/autodiscovery/AccountAutoDiscoveryContent.kt

```diff
@@ -1,80 +1,187 @@
+import app.k9mail.feature.account.common.ui.WizardNavigationBarState
 import app.k9mail.feature.account.common.ui.loadingerror.rememberContentLoadingErrorViewState
 import app.k9mail.feature.account.oauth.ui.AccountOAuthContract
+import app.k9mail.feature.account.oauth.ui.AccountOAuthView
 import app.k9mail.feature.account.oauth.ui.preview.PreviewAccountOAuthViewModel
 import app.k9mail.feature.account.setup.R
 import app.k9mail.feature.account.setup.ui.autodiscovery.AccountAutoDiscoveryContract.Event
 import app.k9mail.feature.account.setup.ui.autodiscovery.AccountAutoDiscoveryContract.State
-import app.k9mail.feature.account.setup.ui.autodiscovery.item.contentItems
+import app.k9mail.feature.account.setup.ui.autodiscovery.view.AutoDiscoveryResultApprovalView
+import app.k9mail.feature.account.setup.ui.autodiscovery.view.AutoDiscoveryResultView
 
 @Composable
 internal fun AccountAutoDiscoveryContent(
     state: State,
     onEvent: (Event) -> Unit,
     oAuthViewModel: AccountOAuthContract.ViewModel,
-    contentPadding: PaddingValues,
     modifier: Modifier = Modifier,
 ) {
-    ResponsiveWidthContainer(
+    Background(
         modifier = Modifier
+            .fillMaxSize()
             .testTag("AccountAutoDiscoveryContent")
-            .padding(contentPadding)
-            .fillMaxWidth()
             .then(modifier),
     ) {
-        val resources = LocalContext.current.resources
+        val scrollState = rememberScrollState()
 
-        ContentLoadingErrorView(
-            state = rememberContentLoadingErrorViewState(state),
-            loading = {
-                LoadingView(
-                    message = stringResource(id = R.string.account_setup_auto_discovery_loading_message),
-                    modifier = Modifier.fillMaxSize(),
-                )
-            },
-            error = {
-                ErrorView(
-                    title = stringResource(id = R.string.account_setup_auto_discovery_loading_error),
-                    message = state.error?.toResourceString(resources),
-                    onRetry = { onEvent(Event.OnRetryClicked) },
-                    modifier = Modifier.fillMaxSize(),
-                )
-            },
-        ) {
-            LazyColumn(
-                modifier = Modifier
-                    .fillMaxSize()
-                    .imePadding(),
-                verticalArrangement = Arrangement.spacedBy(MainTheme.spacings.double, Alignment.CenterVertically),
+        ResponsiveWidthContainer {
+            Column(
+                modifier = Modifier.fillMaxSize(),
             ) {
-                contentItems(
-                    state = state,
-                    onEvent = onEvent,
-                    oAuthViewModel = oAuthViewModel,
+                Column(
+                    modifier = Modifier
+                        .weight(1f)
+                        .verticalScroll(scrollState)
+                        .imePadding(),
```

Comment:
[Optional] I don't think this is doing anything. `FeatureLauncherApp()` already consumes all window insets by using the `safeDrawingPadding()` modifier.

Reply:
With the removed scaffold that is true.